subprojects/resources-http/src/main/java/org/gradle/internal/resource/transport/http/WebdavHttpConnectorFactory.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.internal.resource.transport.http;
+
+import com.google.common.collect.ImmutableSet;
+import org.gradle.authentication.Authentication;
+import org.gradle.authentication.http.BasicAuthentication;
+import org.gradle.authentication.http.DigestAuthentication;
+import org.gradle.authentication.http.HttpHeaderAuthentication;
+import org.gradle.internal.authentication.AllSchemesAuthentication;
+import org.gradle.internal.resource.connector.ResourceConnectorFactory;
+import org.gradle.internal.resource.connector.ResourceConnectorSpecification;
+import org.gradle.internal.resource.transfer.DefaultExternalResourceConnector;
+import org.gradle.internal.resource.transfer.ExternalResourceConnector;
+
+import java.util.Set;
+
+public class WebdavHttpConnectorFactory implements ResourceConnectorFactory {
+    private final static Set<String> SUPPORTED_PROTOCOLS = ImmutableSet.of("dav+http", "dav+https");
```

Comment:
I've spent quite a bit of time attempting to squash insecure protocol use in our ecosystem. 

- https://medium.com/bugbountywriteup/update-want-to-take-over-the-java-ecosystem-all-you-need-is-a-mitm-d069d253fe23
- https://blog.gradle.org/decommissioning-http

I'd consider 'dav+http' an insecure protocol and it should be compatible with the 'allowInsecureProtocol' API.

Reply:
Removed support for 'dav+http'